docs/qlin_genetic/filter_panel/filter_panel.md

```diff
@@ -0,0 +1,23 @@
+# Filter Sidebar
+
+It is possible to filter the data displayed in a table with the help of the Filter Sidebar available to the left of the table.
+
+## Show and Hide
+
+The filter sidebar can be hidden to provide more space for the table by using the icon ![MenuFoldIcon](MenuFoldIcon.png) located at the top. It is also possible to show it again using the icon ![MenuUnfoldIcon](MenuUnfoldIcon.png).
+
+## Filter Types
+
+### Multi-select Filter
+
+This type of filter allows the selection of one or more values using checkboxes. Use the **All** hyperlink to select all possible values. The **None** hyperlink performs the opposite operation. When there are many possible values, only some of them are displayed. A **# More** hyperlink allows you to display all of them using a scroll bar.
+
+A magnifying glass 🔎 located in the filter title allows you to search and filter the list of values.
+
+### Range Filter
+
+This type of filter allows you to specify a minimum value (Less than, Less than or equal to), a maximum value (Greater than, Greater than or equal to), or a range (Between).
```

Comment:
Less than or equal to -> Less than or equal
Greater than or equal to -> Greater than or equal